_source/_posts/2021-12-18-native-java-helidon.adoc

```diff
@@ -0,0 +1,403 @@
+---
+layout: blog_post
+title: "Build REST APIs and Native Java Apps with Helidon"
+author: matt-raible
+by: advocate
+communities: [java]
+description: "Learn how to create REST APIs and native images with Helidon, a Java microservices framework from Oracle."
+tags: [native, java, graalvm, helidon]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+github: https://github.com/oktadev/native-java-examples
+---
+:page-liquid:
+:toc: macro
+:experimental:
+
+Project Helidon is a set of Java libraries for writing microservices. It's an open source, Apache 2.0-licensed, Oracle-sponsored project and contains support for MicroProfile as well as a reactive, functional API. Helidon's two different style are known as Helidon MP and SE, respectively. The https://helidon.io[helidon.io] website says:
```

Comment:
```suggestion
Project Helidon is a set of Java libraries for writing microservices. It's an open source, Apache 2.0-licensed, Oracle-sponsored project and contains support for MicroProfile as well as a reactive, functional API. Helidon's two different styles are known as Helidon MP and SE, respectively. The https://helidon.io[helidon.io] website says:
```
plural

Reply:
I don't know why, but GitHub has the "Commit suggestion" button grayed out.

Reply:
I think this is because this suggestion is on a commit, not on the PR. I'll fix manually.